extensions/extensions-alerting/extensions-alerting-slack/pom.xml

```diff
@@ -22,15 +22,40 @@
             <groupId>org.technologybrewery.krausening</groupId>
             <artifactId>krausening</artifactId>
         </dependency>
-
-        <!-- Logging Dependencies (from aiops-parent pom) -->
         <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-slf4j2-impl</artifactId>
+            <groupId>com.boozallen.aissemble</groupId>
+            <artifactId>foundation-core-java</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.aeonbits.owner</groupId>
+            <artifactId>owner</artifactId>
```

Comment:
Q/A: Was this added because we reference Owner classes via our usage of Krausening? If so, I think it probably makes more sense to just add Krausening and not Owner, as we'd probably not use Owner outside of Krausening.

Reply:
Yes it is and Krausening is already added as a dependency. I have removed the reference to the Owner classes.